models/reading.go

```diff
@@ -34,6 +34,7 @@ type Reading struct {
 	Device   string `json:"device"`
 	Name     string `json:"name"`
 	Value    string `json:"value"` // Device sensor data value
+	Blob     []byte `json:"blob"` // Binary data payload
```

Comment:
Per @tonyespy's [comment](https://github.com/edgexfoundry/go-mod-core-contracts/issues/32#issuecomment-473112193), we need to call this `BinaryValue` instead of `Blob`